spanner/spanner_snippets/spanner/spanner_cancel_backup_create.go

```diff
@@ -20,16 +20,23 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"regexp"
 	"time"
 
 	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	pbt "github.com/golang/protobuf/ptypes/timestamp"
 	"google.golang.org/genproto/googleapis/longrunning"
 	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 func cancelBackup(w io.Writer, db, backupID string) error {
+	matches := regexp.MustCompile("^(.*)/databases/(.*)$").FindStringSubmatch(db)
```

Comment:
Should this be `+` instead of `*`?

```suggestion
	matches := regexp.MustCompile("^(.+)/databases/(.+)$").FindStringSubmatch(db)
```

Reply:
Good point, changed here and in the other two samples.